package/base-files/files/bin/ipcalc.sh

```diff
@@ -50,12 +50,14 @@ BEGIN {
 
 	network=and(ipaddr,netmask)
 	prefix=32-bitcount(compl32(netmask))
-	broadcast=or(network,compl32(netmask))
 
 	print "IP="int2ip(ipaddr)
 	print "NETMASK="int2ip(netmask)
-	print "BROADCAST="int2ip(broadcast)
 	print "NETWORK="int2ip(network)
+	if (prefix<31) {
+		broadcast=or(network,compl32(netmask))
+		print "BROADCAST="int2ip(broadcast)
```

Comment:
The one (packages feed: https://github.com/openwrt/packages/blob/73ee68a5970d1618cdb580001969d13a82e1e76d/net/isc-dhcp/files/dhcpd.init#L353) user I found declares `local BROADCAST` within it's `config_foreach` function, so no issue with using `BROADCAST` of a previous iteration, but it may need adjusting so that it also tests `[ -n $BROADCAST]` if we do not print it.

Reply:
Good find!
I missed that.

This is the PR to address it:
https://github.com/openwrt/packages/pull/21404